app/templates/serviceLearning/slcNewProposal.html

```diff
@@ -32,9 +32,10 @@
       <span class="step"></span>
     </div>
     <div class="col" align="right">
+      <button class="btn btn-primary" type="button" id="saveContinue">Save and Continue</button>
       <button class="btn btn-primary" type="button" id="nextButton">Next</button>
       {% if g.current_user.isCeltsAdmin %}
-        <button class="btn btn-success" type="button" id="saveAndApproveButton">Save and Approve</button>
+        <button class="btn btn-success" type="button" id="approveButton">Save and Approve</button>
```

Comment:
Let's rename this "Submit and Approve", to make it clearer that it is the same kind of thing as Submit